# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libjar/test/unit/test_jarchannel.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c modules/libjar/test/unit/test_jarchannel.js
--- a/modules/libjar/test/unit/test_jarchannel.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/modules/libjar/test/unit/test_jarchannel.js	Tue Feb 12 16:08:25 2019 +0000
@@ -76,7 +76,7 @@
     var uri = jarBase + "/inner40.zip";
     var chan = NetUtil.newChannel({uri: uri, loadUsingSystemPrincipal: true});
     Assert.ok(chan.contentLength < 0);
-    chan.asyncOpen2(new Listener(function(l) {
+    chan.asyncOpen(new Listener(function(l) {
         Assert.ok(chan.contentLength > 0);
         Assert.ok(l.gotStartRequest);
         Assert.ok(l.gotStopRequest);
@@ -112,7 +112,7 @@
 add_test(function testSync() {
     var uri = jarBase + "/inner40.zip";
     var chan = NetUtil.newChannel({uri: uri, loadUsingSystemPrincipal: true});
-    var stream = chan.open2();
+    var stream = chan.open();
     Assert.ok(chan.contentLength > 0);
     Assert.equal(stream.available(), chan.contentLength);
     stream.close();
@@ -128,7 +128,7 @@
 add_test(function testSyncNested() {
     var uri = "jar:" + jarBase + "/inner40.zip!/foo";
     var chan = NetUtil.newChannel({uri: uri, loadUsingSystemPrincipal: true});
-    var stream = chan.open2();
+    var stream = chan.open();
     Assert.ok(chan.contentLength > 0);
     Assert.equal(stream.available(), chan.contentLength);
     stream.close();
@@ -143,7 +143,7 @@
 add_test(function testAsyncNested(next) {
     var uri = "jar:" + jarBase + "/inner40.zip!/foo";
     var chan = NetUtil.newChannel({uri: uri, loadUsingSystemPrincipal: true});
-    chan.asyncOpen2(new Listener(function(l) {
+    chan.asyncOpen(new Listener(function(l) {
         Assert.ok(chan.contentLength > 0);
         Assert.ok(l.gotStartRequest);
         Assert.ok(l.gotStopRequest);
@@ -163,7 +163,7 @@
     file.copyTo(copy.parent, copy.leafName);
     var uri = "jar:" + ios.newFileURI(copy).spec + "!/inner40.zip";
     var chan = NetUtil.newChannel({uri: uri, loadUsingSystemPrincipal: true});
-    var stream = chan.open2();
+    var stream = chan.open();
     Assert.ok(chan.contentLength > 0);
     stream.close();
 
@@ -192,7 +192,7 @@
     var uri = "jar:" + ios.newFileURI(copy).spec + "!/inner40.zip";
     var chan = NetUtil.newChannel({uri: uri, loadUsingSystemPrincipal: true});
 
-    chan.asyncOpen2(new Listener(function (l) {
+    chan.asyncOpen(new Listener(function (l) {
         Assert.ok(chan.contentLength > 0);
 
         // Drop any jar caches